pyproject.toml

```diff
@@ -94,7 +93,7 @@ ensure_newline_before_comments = true
 line_length = 88
 
 [tool.mypy]
-python_version = 3.9
+python_version = 3.10
```

Comment:
I am wondering if quotes would not be required here to avoid being loaded as 3.1 float!